### ISSUE TITLE
Crash on startup with error: "GPU process isn't usable. Goodbye."

### ISSUE THREAD

**Issue by murnux** [Heroic-Games-Launcher/HeroicGamesLauncher#666]
**Describe the bug**
Opening Heroic on OpenSUSE Tumbleweed is resulting in a crash with the following error (error is viewable when running from the terminal): 
```
% heroic
...
[24203:0925/110558.796825:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
```
When opening Heroic from my desktop environment, the Heroic window briefly opens, then immediately closes.

Here's what I've done to try and fix the issue:

1. Searched around closed / open issues, didn't see any other mention of this error.
2. Tried deleting the `~/.config/heroic` directory per the wiki.
3. Confirmed Python install was correct, though this is probably an Electron problem.
4. I've tried installing by both building an RPM of the app myself, and using the AppImage. Same issues happens with both methods.

**To Reproduce**

1. From a terminal, run: `heroic`.
2. Error appears

**Expected behavior**
I would expect the application to launch as expected without errors.

**Desktop (please complete the following information):**

- OS: OpenSUSE Tumbleweed (fully updated)
- DE: Gnome 40.4
- Hardware specs (RAM, CPU, GPU):
Kernel 5.14.5
32 GB of RAM
GTX 1070, Nvidia Driver 470.74
AMD Ryzen 2700

**Additional context**
I'm using Visual Studio Code and Discord on this machine, both of which are Electron, so I doubt it's a general Electron problem.

**Comment by flavioislima** [comment]
Hi.

Are you on wayland?

**Comment by murnux** [comment]
Hello, thanks for the quick response. I'm on X11.

```
➜  ~ echo $XDG_SESSION_TYPE
x11
```

**Comment by toalex77** [comment]
As a workaround, you can try to run Heroic with flag --disable-gpu-sandbox.

**Comment by flavioislima** [comment]
> As a workaround, you can try to run Heroic with flag --disable-gpu-sandbox.

Yes, I was trying to remember that flag. If it works I'll add this info on our Wiki. thanks for that.
@murnux  let us know if it worked.

**Comment by murnux** [comment]
Running the `--disable-gpu-sandbox` did the trick. Thanks for the assistance!

**Comment by flavioislima** [comment]
Found out this is a Electron issue related with a Glibc release. It was fixed already but we will need to update Heroic, so this will be fixed for good on next version.

Reference: https://forums.opensuse.org/showthread.php/560226-Electron-apps-fail-to-launch-FATAL-gpu_data_manager_impl_private-cc(415)-GPU-process-isn-t-usable

**Comment by murnux** [comment]
For general reference, this was a much larger problem with tumbleweed; they upgraded `glibc` and a lot of various apps broke. I believe the only solution to rollback to the old glibc is to rollback using a snapshot, which most OpenSUSE installs provide.
